.github/workflows/compose-full-changelog.yaml

```diff
@@ -15,6 +15,12 @@ jobs:
       - name: Checkout
         uses: actions/checkout@v2
 
+      - name: Install Node.js 14.x
+        uses: actions/setup-node@v1
+        with:
+          node-version: '14.x'
```

Comment:
I suggest defining this in [an `env` key](https://docs.github.com/en/actions/using-workflows/workflow-syntax-for-github-actions#env) at the top of the file, as is the established convention in the Tooling Team workflows ([example](https://github.com/arduino/tooling-project-assets/blob/main/workflow-templates/test-go-task.yml#L4-L6))

```yaml
env:
  # See: https://github.com/actions/setup-node/#readme
  NODE_VERSION: 14.x
```
[...]
```
      - name: Install Node.js
        uses: actions/setup-node@v1
        with:
          node-version: ${{ env.NODE_VERSION }}
```
The reason for this is it makes maintenance easier when we need to update tool versions.